modules/ROOT/pages/box/box-connector-examples.adoc

```diff
@@ -250,6 +250,260 @@ output application/json
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Working Dircetory | ${mule.home}/apps/${app.name}/
+|===
++
+. Drag a Read File operation onto canvas and configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name |Read File
+|Connector configuration | Select the File Configuration that had been created previously in Global Elements
+|File Path |`input/yourFileName`
+|===
++
+. Now you need to transform this payload to multipart/form-data type of request. You can also specify a folder to which will be file uploaded vby setting the ID of parent folder. Drag a DataWeave component next to the Read File component and use the script below.
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output multipart/form-data
+---
+{
+  parts : { 
+  	attributes : { 
+  			headers : {
+        		"Content-Type": "application/json"},
+      		content : {
+      			"name": message.attributes.fileName ,
+	// Set the ID of the parent folder. Use "0" for the root folder.
+      	 		"parent": {
+      	 		  "id":"0"
+				}
+			}
+	},
+    file : {
+      headers : {
+        "Content-Disposition" : {
+            "name": "file",
+            "filename": message.attributes.fileName
+        }
+      },
+      content : payload
+    }
+  }
+}
+----
++
+. Configure the HTTP Request by adding a new HTTP Request Global Element. Click on the Global Elements tab at the base of the canvas. In the Global Configuration Elements screen, click Create. In the Choose Global Type wizard, expand Connector Configuration and select HTTP Request Configuration and click Ok. In the HTTP Request Configuration screen configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Name| HTTP_Request_configuration
+|Base path |/
+|Protocol | HTTP(Default)
+|Host |80
+|Port |
+|Use persistent connections| ✓
+|Max Connections|-1
+|Connection idle timeout|30000
+|Response buffer size|1024
+|TLS configuration| None
+|Proxy|None
+|Authentication|None
+|Client socket properties|None
+|Use reconnection| unchecked
+|===
++
+. Now drag a HTTP request onto canvas and configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name| Request - Box Upload File
+|Configuration| Select the HTTP request configuration that had been created previously in Global Elements
+|Method|POST
+|Path|
+|URL|https://upload.box.com/api/2.0/files/content
+|Body|payload
+|===
++
+. Now click we need to create Authorization header in the HTTP Request. Click on the Headers tab next to the Body in the HTTP request , click on the plus sign and configure the following parameter:
```

Comment:
Rewrite to: 
Next create an Authorization header in the HTTP Request. Click the *Headers* tab next to the Body in the HTTP request, then click the plus sign and configure the following parameter:

Reply:
Rewrote